src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1320,22 +1320,168 @@
   "com.minecolonies.coremod.research.havetounlock": "Build a University and research to unlock this Building!",
 
   "com.minecolonies.coremod.info.builder.0.name": "Introduction",
-  "com.minecolonies.coremod.info.builder.0": "The builder is the heart of your colony, without it you wouldn't ever be able to get anything done!\n Every hut worker has to have their building built to at least level 1 before they can do anything, except the builder, the builder at level 0 (no hut) is able to build one thing, their own hut.\n\n In order for your builder to build or upgrade a hut that isn't their own, their hut must be at least the same level as the other hut, e.g. their hut must be level 3 or above to build a level 3 hut.",
+  "com.minecolonies.coremod.info.builder.0": "The Builder is the heart of your colony. Without them you wouldn't be able to get anything done!\n Every hut worker has to have their building built to at least level 1 before they can do anything, except the Builder. If the Builder has a hut at level 0 (no hut), they can only build one thing, their own hut.\n\n For your Builder to build, upgrade, reposition, or repair a hut that isn't their own, their hut must be at least the same level as the other hut. For example, the Builder's Hut must be level 3 or above to build a level 3 hut.",
```

```suggestion
  "com.minecolonies.coremod.info.builder.0": "The Builder is the heart of your colony. Without them, you wouldn't be able to get anything done!\n Every hut worker has to have their building built to at least level 1 before they can do anything, except the Builder. If the Builder has a hut at level 0 (no hut), they can only build one thing, their own hut.\n\n For your Builder to build, upgrade, reposition, or repair a hut that isn't their own, their hut must be at least the same level as the other hut. For example, the Builder's Hut must be level 3 or above to build a level 3 hut.",
```

oops, I forgot this comma